tests/unittests/CoreImage/CIContextTests.mm

```diff
@@ -18,33 +18,61 @@
 
 #import <UIKit/UIKit.h>
 #import <CoreImage/CoreImage.h>
-#import "Starboard/SmartTypes.h"
+#import <Starboard/SmartTypes.h>
 #import "CALayerInternal.h"
 #import "../UIKit/NullCompositor.h"
+#import <CoreGraphics/CGImage.h>
+#import <ImageIO/ImageIO.h>
 
 #include <windows.h>
 
-DISABLED_TEST(CoreImage, CGImageFromRect) {
+static NSString* getPathToFile(NSString* fileName) {
+    static StrongId<NSString*> refPath = []() {
+        char fullPath[_MAX_PATH];
+        GetModuleFileNameA(NULL, fullPath, _MAX_PATH);
+        return [@(fullPath) stringByDeletingLastPathComponent];
+    }();
+    return [refPath stringByAppendingPathComponent:fileName];
+}
+
+static CFMutableDataRef ObtainJPEGRepresentationFromCGImage(CGImageRef image) {
```

Comment:
this is getting removed with the UIImage JPEG representation and this works :D
So leaving it.

---
In reply to: [95705999](https://github.com/Microsoft/WinObjC/pull/1625#discussion_r95705999) [](ancestors = 95705999)